addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_data.xml

```diff
@@ -417,231 +417,231 @@
-            <field name="reconcile" eval="False" />
+            <field name="reconcile" eval="True" />
             <field name="user_type_id" ref="account.data_account_type_payable" />
             <field name="chart_template_id" ref = "l10n_chart_china_small_business" />
           </record>
           <record id="small_business_char2211003" model="account.account.template">
             <field name="code">2211003</field>
             <field name="name">社会保险费</field>
-            <field name="reconcile" eval="False" />
+            <field name="reconcile" eval="True" />
             <field name="user_type_id" ref="account.data_account_type_payable" />
             <field name="chart_template_id" ref = "l10n_chart_china_small_business" />
           </record>
           <record id="small_business_char2211004" model="account.account.template">
             <field name="code">2211004</field>
             <field name="name">住房公积金</field>
-            <field name="reconcile" eval="False" />
+            <field name="reconcile" eval="True" />
             <field name="user_type_id" ref="account.data_account_type_payable" />
             <field name="chart_template_id" ref = "l10n_chart_china_small_business" />
           </record>
           <record id="small_business_char2211005" model="account.account.template">
             <field name="code">2211005</field>
             <field name="name">工会经费</field>
-            <field name="reconcile" eval="False" />
+            <field name="reconcile" eval="True" />
             <field name="user_type_id" ref="account.data_account_type_payable" />
             <field name="chart_template_id" ref = "l10n_chart_china_small_business" />
           </record>
           <record id="small_business_char2211006" model="account.account.template">
             <field name="code">2211006</field>
             <field name="name">职工教育经费</field>
-            <field name="reconcile" eval="False" />
+            <field name="reconcile" eval="True" />
             <field name="user_type_id" ref="account.data_account_type_payable" />
             <field name="chart_template_id" ref = "l10n_chart_china_small_business" />
           </record>
           <record id="small_business_char2211007" model="account.account.template">
             <field name="code">2211007</field>
             <field name="name">解除职工劳动关系补偿</field>
-            <field name="reconcile" eval="False" />
+            <field name="reconcile" eval="True" />
             <field name="user_type_id" ref="account.data_account_type_payable" />
             <field name="chart_template_id" ref = "l10n_chart_china_small_business" />
           </record>
           <record id="small_business_char2221" model="account.account.template">
             <field name="code">2221</field>
             <field name="name">应交税费</field>
-            <field name="reconcile" eval="False" />
+            <field name="reconcile" eval="True" />
```

Comment:
why ？

Reply:
if this is set to true, when you pay for invoice with tax, you will get a error message:  
"Entries are not from the same account."

that is the reason why I need this new version of l10n_cn accounts modules been merge to version 12 ASAP! @qdp-odoo